deepchem/utils/save.py

```diff
@@ -23,11 +27,63 @@ def log(string, verbose=True):
     print(string)
 
 
-def save_to_disk(dataset, filename, compress=3):
+def save_to_disk(dataset, filename, compress=0):
   """Save a dataset to file."""
+
   joblib.dump(dataset, filename, compress=compress)
 
 
+def save_sparse_mats(mat_b, filename):
```

Comment:
@rbharath New csr format for storing sparse matrices. Turns out numpy arrays are binary appendable, which made this super easy to roll out.